protocol/agents.json

```diff
@@ -234,6 +234,14 @@
                 "ably-go"
             ],
             "type": "sdk"
+        },
+        {
+            "identifier": "go",
+            "versioned": true,
+            "emitters": [
+              "ably-go"
```

Comment:
Please use a 4-space indent

Reply:
I don't have indentation tool : (
Seems need to do it manually

Reply:
Fixed !